packages/eslint-plugin-template/src/rules/no-duplicate-attributes.ts

```diff
@@ -0,0 +1,89 @@
+import { TmplAstElement } from '@angular/compiler';
+import {
+  createESLintRule,
+  getTemplateParserServices,
+} from '../utils/create-eslint-rule';
+import {
+  BoundAttribute,
+  BoundEvent,
+  TextAttribute,
+} from '@angular/compiler/src/render3/r3_ast';
+
+type Options = [];
+export type MessageIds = 'noDuplicateAttributes';
+export const RULE_NAME = 'no-duplicate-attributes';
```

Comment:
The name could probably use some bike shedding here, I couldn't really think of much else though. An input and output on a component are technically html attributes so I don't think it's an incorrect name to use but maybe someone else can think of a better name?

Reply:
If this only deals with Angular's Inputs and Outputs, rather than all HTML attributes, would `no-duplicate-bindings` be more appropriate?

Reply:
It does also deal with all html attributes e.g. if you did `<input type="text" type="number">` it would get caught by this rule. Would `no-duplicate-bindings-or-attributes` be too verbose as a rule name?

Reply:
Ah ok cool. Yeah it's a tricky one - correctness vs terseness... The Angular syntax binds to properties, not attributes, so it's technically missing out that distinction, but how much does that matter in practice? Probably not a lot.

Let's proceed as is for now with the more terse name, I don't think there is one term that technically works for attributes, bindings and properties.